### DISCUSSION TITLE
Prometheus is offline abnormally

### DISCUSSION THREAD

**Discussion by CrushLSG** [strimzi/strimzi-kafka-operator#6404] [Q&A]
In my domain, there is such a problem: in Prometheus, an OS is actually restarted abnormally and offline. It is notified in alertmanager that multiple OSS are offline. Under the status in Prometheus, there is actually one target, but the notification is that there are two, which is very strange

It is confirmed that the problem is not caused by cumulative notification
![Snipaste_2022-02-22_14-27-18](https://user-images.githubusercontent.com/77657457/155075171-b7d9b971-b617-4e46-b539-e39da7833bdd.png)

**Comment by scholzj** [comment]
Sorry, how does this relate to Strimzi?
